modules/fb_instant_articles_display/src/DrupalInstantArticleDisplay.php

```diff
@@ -62,7 +62,7 @@ class DrupalInstantArticleDisplay {
    *
    * @var InstantArticle
    */
-  private $instantArticle;
+  public $instantArticle;
```

Comment:
Essentially because we need it in this line: https://github.com/BurdaMagazinOrg/module-fb_instant_articles/pull/29/files#diff-26b2d4f4256779e153e253fbc312d48fR71

Makes sense?